data/json/effects_on_condition/misc_effect_on_condition.json

```diff
@@ -278,13 +278,58 @@
     "run_for_npcs": true,
     "condition": { "and": [ "u_is_outside", "u_can_see" ] },
     "effect": [
-      { "u_message": "The sunrise always makes you a feel a little bit better." },
       {
-        "u_add_morale": "morale_sunrise",
-        "bonus": 10,
-        "max_bonus": 10,
-        "duration": "120 minutes",
-        "decay_start": "60 minutes"
+        "if": { "or": [ { "is_weather": "sunny" }, { "is_weather": "clear" } ] },
+        "then": [
+          {
+            "if": { "and": [ { "u_has_trait": "SPIRITUAL" }, { "x_in_y_chance": { "x": 1, "y": 2 } } ] },
+            "then": [
+              { "u_message": "You can feel God's doing in this very beautiful sunrise." },
```

Comment:
Spiritual isn't necessarily someone who is into capital G god, could this be reworded in a more generically spiritual way? Same goes for the other one where it's used